README.md

```diff
@@ -2,44 +2,145 @@
+<!-- Read SOPO: look for nouns in the relevant task descriptions. -->
+
+
 The API for fit, predict, and metrics is reduced to specifying a start and end times for a given location.
 The model must construct feature data using column transforms. Having done so, forecasting as a service become trivial.
 
 ## Installation
 
-requires the eccodes library: `apt-get install libeccodes-dev`
+<!-- Talk about env for this to run - should be a complete description - provide the context. -->
 
+This library is designed for use by technical engineers and data scientists. It takes advantage of the python
+data science ecosystem and therefore requires installation of many third party open source libraries. It has 
+been developed and tested in a linux operating system. Running on a docker container such as 
+the [canonical Ubuntu image](https://hub.docker.com/_/ubuntu) is strongly recommended. The library was 
+developed using Ubuntu 22.04 (Jammy) with Python 3.10.6.
+
+### Installing system libraries
+Reading the grib weather data requires the eccodes library: `apt-get install libeccodes-dev`
+
+### Installing python libraries
 `pip3 install -r requirements.txt`
 
 Verify you can run the unit tests `python -m unittest`
 
-If running in a container, to start a notebook use `jupyter notebook --NotebookApp.ip=0.0.0.0`
+To start jupyter notebook run `jupyter notebook --NotebookApp.ip=0.0.0.0`
+Now you can open the example notebook and execute the cells in the demonstration.
+
+<!-- Phase 2 (after FTR accepted): add docker container in July -->
 
 ## Usage
-Models can be composed of mixins for various estimators and forecast processes.
+Models can be composed of mixins for various estimators and forecast processes. These composable
+pieces can be put together in different ways to solve many problems. The RegularTimeSeriesModel is the base.
+The core that problem specific parts are added to when forecasting or gap filling a particular timeseries.
+The estimator is the next essential building block. It can be a Classifier or a Regressor. There are many 
+different numerical techniques that can be applied via the estimator. The process is the last essential 
+component. It defines the timeseries being forecast and the available feature data that might have predictive value.
+Having composed a Model class from these three parts, it is then up to the user to create an instance of the class that 
+with configuration arguments that tune the model features for the specific meter load or pv forecast. 
+
 
-An ordinary least squares model for a Balancing Area Hourly Load
+<!-- Explain what this does - for lawyers... point to ipynb example -->
+<!-- Consider adding sklearn model diagram if it would help (rather than hurt) the explanation -->
+
+### Compose a model class
+
+A model using an Ordinary Least Squares estimator to forecast a Balancing Area Hourly Load
 ```python
 class NpOlsModel(BalancingAreaHourly, LinearRegressor, RegularTimeSeriesModel):
   pass
 ```
```

Comment:
````suggestion
New models are defined as Python classes, which utilize building blocks provided by this library as base classes. For example, here is the beginning of a model using an Ordinary Least Squares estimator to forecast Balancing Area Hourly Load:

```python
class NpOlsModel(BalancingAreaHourly, LinearRegressor, RegularTimeSeriesModel):
  ...
```
````

Also maybe explain what "NP" is.

Also also, is an `import` statement necessary here, that we should include for completeness?

P3